hypertools/_shared/helpers.py

```diff
@@ -136,6 +136,44 @@ def reshape_data(x,labels):
         x_reshaped[categories.index(point)].append(x_stacked[idx])
     return [np.vstack(i) for i in x_reshaped]
 
+def is_label_probabilistic(labels):
+    """
+        Tells Whether the Assigned Label is a Scalar or Not[Case for GaussianMixture models]
+    """
+    if type(labels[0]) is np.ndarray or type(labels[0]) is list:
```

Comment:
we should also check whether `labels.shape[1] > 1` to support pandas `DataFrame` objects as labels.  Alternatively, we could call `labels = hyp.tools.format_data(labels)` before line 142, or before this function is called.

Reply:
I will check it and modify the testing condition.

Reply:
Added a Check for DataFrame: `if type(labels[0]) is np.ndarray or type(labels) is pd.DataFrame or type(labels[0]) is list`